ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Change Log - oav
 
+## 06/03/2024 3.3.7
```

Comment:
I didn't publish 3.3.6, so unless you did, I think we can re-use 3.3.6 for the combined changes.

Reply:
Absolutely right.